src/runmode-af-packet.c

```diff
@@ -290,7 +295,9 @@ static void *ParseAFPConfig(const char *iface)
     if (ConfGetChildValueWithDefault(if_root, if_default, "cluster-id", &tmpclusterid) != 1) {
         aconf->cluster_id = (uint16_t)(cluster_id_auto++);
     } else {
-        aconf->cluster_id = (uint16_t)atoi(tmpclusterid);
+        if (ByteExtractStringInt32(&aconf->cluster_id, 10, 0, (const char *)tmpclusterid) < 0) {
+            goto finalize;
```

Comment:
Should we set `cluster_id = 0` ?

Reply:
Yes, maybe. goto is making it skip every other config.